### PR TITLE
Regenerate with latest gir/gir-files

### PR DESCRIPTION
**Pull request by bilelmoussaoui**
Contains some updated bindings as well, haven't reviewed carefully yet

### CODE REVIEW

File: pango/src/auto/item.rs [hunk 1]
```diff
@@ -33,6 +33,14 @@ impl Item {
         }
     }
 
+    #[cfg(feature = "v1_54")]
+    #[cfg_attr(docsrs, doc(cfg(feature = "v1_54")))]
+    #[doc(alias = "pango_item_get_char_offset")]
+    #[doc(alias = "get_char_offset")]
+    pub fn char_offset(&mut self) -> i32 {
```
Comment (bilelmoussaoui) [inline pango/src/auto/item.rs:40]: should be 
```suggestion
    pub fn char_offset(&self) -> i32 {
```

File: pango/sys/Cargo.toml [hunk 2]
```diff
@@ -21,6 +21,7 @@ v1_46 = ["v1_44"]
 v1_48 = ["v1_46"]
 v1_50 = ["v1_48"]
 v1_52 = ["v1_50"]
+v1_54 = ["v1_52"]
```
Comment (bilelmoussaoui) [inline pango/sys/Cargo.toml:24]: needs a feature in pango/Cargo.toml + CI update

File: pango/src/auto/item.rs (hunk 1 above)
Comment (sdroege) [reply pango/src/auto/item.rs]: Can be configured in `Gir.toml`

Comment (bilelmoussaoui) [reply pango/src/auto/item.rs]: yes, that is what i did

**Review by sdroege — APPROVED** [review]

**Pull request merged**